docs/tutorial.md

```diff
@@ -1,3 +1,117 @@
-# Tutorial
+# Tutorial: using the AWS plugins for Backstage
 
-Coming soon!
+This tutorial walks you through an example of using the AWS plugins for Backstage.
+
+Tutorial steps:
+
+<!-- toc -->
+
+1. [Set up your Backstage app](#set-up-your-backstage-app)
+1. [Clone this repository](#clone-this-repository)
+1. [Create prerequisite AWS resources](#create-prerequisite-aws-resources)
+1. [Run the Backstage app locally](#run-the-backstage-app-locally)
+1. [Explore the Plugins](#explore-the-plugins)
+1. [Cleanup](#cleanup)
+<!-- tocstop -->
+
+## Set up your Backstage app
+
+This tutorial assumes that you have a working Backstage application. To use this tutorial with a local Backstage app, follow the main [Backstage instructions](https://backstage.io/docs/getting-started/create-an-app) to create and run a Backstage app locally.
+
+## Clone this repository
+
+Before we start, clone the repository to get access to the necessary sample code and configurations.
+
+```
+git clone https://github.com/awslabs/backstage-plugins-for-aws.git
+cd backstage-plugins-for-aws
+```
+
+## Create prerequisite AWS resources
+
+To get started, we'll deploy a sample AWS CDK project included in the repository. The CDK project simplifies setup by automatically creating sample ECS services and other resources. These resources allow you to interact with real data in the Backstage plugins without needing to configure everything manually.
```

Comment:
Expand on this slightly to explain that it will create an ECS cluster, ECS service and a CodePipeline which builds and deploys a container image.